### ISSUE TITLE
Python 3.x: Can't import version file when running okta-awscli

### ISSUE THREAD

**Issue by dougireton** [jmhale/okta-awscli#38]
Inside a Python 3.6.5 virtualenv:

```
okta-awscli
Traceback (most recent call last):
  File "/usr/local/bin/okta-awscli", line 7, in <module>
    from oktaawscli.okta_awscli import main
  File "/usr/local/lib/python3.6/site-packages/oktaawscli/okta_awscli.py", line 4, in <module>
    from version import __version__
ModuleNotFoundError: No module named 'version'
```

**Comment by dougireton** [comment]
This works with Python 2.7.10.

**Comment by brianwallace** [comment]
From what I can tell, this particular ` from version import __version__` import error already seems fixed in `master`, but I found some additional python 3.6.x issues which I fixed in the following PR:

https://github.com/jmhale/okta-awscli/pull/39

**Comment by jmhale** [comment]
This is confirmed fixed in #39 (and cut a release in #40). PyPi has been updated to v0.2.2 as well.

Thanks @brianwallace!
